### PR TITLE
[DOCS] Add setup content to Kubernetes and Cloud Foundry docs

### PR DESCRIPTION
**Pull request by EamonnTP**
## Summary

This PR adds the setup content to both the Kubernetes and Cloud Foundry docs for Filebeat and Metricbeat.

The PR also adds the tabbed widgets to the Load Kibana dashboards topic. 

**NOTE**: In a follow-up PR, I plan to single-source the Kubernetes content for both the Filebeat and Metricbeat docs. 

### HTML preview

**Run Filebeat on Cloud Foundry**: https://beats_23580.docs-preview.app.elstc.co/guide/en/beats/filebeat/master/running-on-cloudfoundry.html

**Run Metricbeat on Cloud Foundry**: https://beats_23580.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/running-on-cloudfoundry.html

**Run Filebeat on Kubernetes**: https://beats_23580.docs-preview.app.elstc.co/guide/en/beats/filebeat/master/running-on-kubernetes.html

**Run Metricbeat on Kubernetes**: https://beats_23580.docs-preview.app.elstc.co/guide/en/beats/metricbeat/master/running-on-kubernetes.html

**Load Kibana dashboards**: https://beats_23580.docs-preview.app.elstc.co/guide/en/beats/auditbeat/master/load-kibana-dashboards.html

## Related issue

Closes https://github.com/elastic/beats/issues/22772

### CODE REVIEW

**Review by jsoriano — APPROVED** [review]
Great, thanks!

File: libbeat/docs/shared-cloudfoundry.asciidoc [hunk 1]
```diff
@@ -53,6 +53,42 @@ curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/cloud
 You need to modify the +{beatname_lc}.yml+ file to set the `api_address`,
 `client_id` and `client_secret`.
 
+==== Load {kib} dashboards
+
+{beatname_uc} comes packaged with example {kib} dashboards,
```
[the remark this answers is not in the document]
Comment (dedemorton) [reply libbeat/docs/shared-cloudfoundry.asciidoc:58]: You can see what I did in the k8s monitoring guide: https://www.elastic.co/guide/en/observability/master/monitor-kubernetes.html#_out_of_the_box_kibana_dashboards (though of course I'm open to feedback if you can think of something better).

File: metricbeat/docs/running-on-kubernetes.asciidoc [hunk 2]
```diff
@@ -169,6 +169,29 @@ oc patch namespace kube-system -p \
 This command sets the node selector for the project to an empty string. If you
 don't run this command, the default node selector will skip master nodes.
 
+[float]
+==== Load {kib} dashboards
```
Comment (dedemorton) [inline metricbeat/docs/running-on-kubernetes.asciidoc:173]: Same comments as I made for CloudFoundry apply here: too much duplicated text.

Another consideration is that k8s users won't have the beats package downloaded to their local machine, so they'll need access to those steps.

File: filebeat/docs/running-on-kubernetes.asciidoc [hunk 3]
```diff
@@ -112,6 +112,36 @@ oc patch namespace kube-system -p \
 This command sets the node selector for the project to an empty string. If you
 don't run this command, the default node selector will skip master nodes.
 
+[float]
```
Comment (dedemorton) [inline filebeat/docs/running-on-kubernetes.asciidoc:115]: See my comments about CloudFoundry and running Metricbeat on k8s. They apply here, too.

File: libbeat/docs/howto/load-dashboards.asciidoc [hunk 4]
```diff
@@ -15,98 +15,44 @@
...
-----------------------------------------------------------------------
-
-*brew:*
-
-["source","sh",subs="attributes"]
-----------------------------------------------------------------------
-{beatname_lc} setup --dashboards
-----------------------------------------------------------------------
-endif::mac_os[]
+include::{libbeat-dir}/tab-widgets/load-dashboards-widget.asciidoc[]
```
Comment (dedemorton) [inline libbeat/docs/howto/load-dashboards.asciidoc:48]: Awesome! Thank you so much for adding these widgets.

**Review by dedemorton — COMMENTED** [review]
Thanks again for adding all those widgets!

**Comment by EamonnTP** [comment]
@dedemorton Thanks for your comments! I tweaked the content a little.

File: libbeat/docs/shared-cloudfoundry.asciidoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (EamonnTP) [reply libbeat/docs/shared-cloudfoundry.asciidoc]: I agree. Thanks!

**Review by dedemorton — APPROVED** [review]
Thanks! Changes LGTM.

**Comment by EamonnTP** [comment]
jenkins run the tests please

**Pull request merged**